### ISSUE TITLE
Set language dynamically

### ISSUE THREAD

**Issue by Alexey6486** [isaachinman/next-i18next#901]
Hello.
I'm trying to set a language depends on a user preference. 
While a page is loading I get user's language settings in MyApp.getInitialProps method and want to use it as a defaultLanguage param for next-i18Next instance. 
I tried to do something like config.defaultLanguage or i18n.changeLanguage in MyApp.getInitialProps, but to no avail.
The code is exactly like in the simple example and it works fine if languages are set in the new NextI18Next instance.
But if I want to set them later, it doesn't work... at least I didn't succeeded in it.
Appreciate any help.

**Comment by kachkaev** [comment]
The general direction of this library is to align with Next.js i18n feature. See https://github.com/isaachinman/next-i18next/issues/869 and https://nextjs.org/docs/advanced-features/i18n-routing. So changing the language ‘dynamically’ will be possible as much as Next.js itself allows.

**Comment by Alexey6486** [comment]
> The general direction of this library is to align with Next.js i18n feature. See #869 and https://nextjs.org/docs/advanced-features/i18n-routing. So I guess that changing the language ‘dynamically’ will be possible as much as Next.js itself allows.

Thank you for answering me on such a short notice.
I've made myself acquainted with information provided by the links. Unfortunately, I didn't find exactly what I'm looking for. 

In the simple example Isaac creates a Next-i18Next instance in the i18n.js file. And the app language with dictionary is loaded according to the settings from the file. But can we set language at MyApp.getInitialProps stage (provided we do not use any locale params of language in url)? This is what I'm trying to figure out.
